app/Services/EntityService.php

```diff
@@ -657,7 +657,14 @@ public function makeNewMentionEntity(MiscModel $model, string $name)
         $model->forceSavedObserver = true;
         $model->is_private = $defaultPrivate;
         $model->save();
-
+        if ($model->entity->type() != 'tag') {
```

Comment:
This works, but we have a slightly faster option that will compare numbers rather than strings.
`$model->entity->type_id !== config('entities.ids.tag')`